### ISSUE TITLE
BuildFakeEmailAddress() doesn't produce valid address when name ends with full stop

### ISSUE THREAD

**Issue by phw198** [phw198/OutlookGoogleCalendarSync#1010]
One of attendee address of the event has an error (extra dot at the end XXX@YYY.com.) and ogcs keep rising error because the email is invalid in google sync.

Cannot act on the meeting because I am not the organizer, thus I have a crash every 1 hour (sync periodicity).

Have made a workaround using specific category and exclusion filter but not quite good in my opinion.

Version of ogc: `2.8.0.0 `
Log

```
2020-06-02 17:43:26,385 DEBUG  25 OutlookGoogleCalendarSync.EmailAddress [0] -  Built a fake email for them: marion.schenkweinvayla.fi.@unknownemail.com
2020-06-02 17:43:26,387 ERROR  25 OutlookGoogleCalendarSync.Console [0] -  Recipient "marion.schenkwein@vayla.fi." with email address "marion.schenkweinvayla.fi.@unknownemail.com" is invalid.
2020-06-02 17:43:26,389 ERROR  25 OutlookGoogleCalendarSync.Console [0] -  This must be manually resolved in order to sync this appointment.
2020-06-02 17:43:26,537 WARN  25 OutlookGoogleCalendarSync.Console [0] -  04/06/2020 14:00  "GoToMeeting Invitation - IFC Rail Call for Participation /Open Forum"
2020-06-02 17:43:26,537 WARN  25 OutlookGoogleCalendarSync.Console [0] -  Event creation skipped.
```

_Originally posted by @MatthieuPERIN in https://github.com/phw198/OutlookGoogleCalendarSync/issues/7#issuecomment-637999870_

**Comment by phw198** [comment]
:package: This should now be resolved with [v2.8.3.19.zip](https://github.com/phw198/OutlookGoogleCalendarSync/files/4777124/v2.8.3.19.zip) ([see instructions for applying](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-Problems#applying-a-hotfix)).

Please let me know how it goes.

**Comment by MatthieuPERIN** [comment]
**Preliminaries** 
Upgrades to alpha + Apply Patch ✔
**Test**
Create a dummy appointment with an attendee with extra full stop in e-mail address ✔
Synch without crash ✔ 🥳 🍾 
Name with the extra dot is kept in GC name of attendee but adresse is corrected ✔
**Result**
Perfect ! thanks for the help and quick fix 👍
